implementation/data_filter.py

```diff
@@ -1,33 +1,63 @@
-# Data Filtering 
-from judge import DirectAssessment
 import dspy
-from dspy import Prediction
+from judge import DirectAssessment
+from prometheus_eval.prompts import SCORE_RUBRIC_TEMPLATE
 
 class DataFilter(dspy.Module):
     def __init__(self):
         super().__init__()
-        self.judge = DirectAssessment()
-        
-
 
 class ResponseFilter(DataFilter):
+    def __init__(self, model, score_threshold=4):
```

Comment:
I'm not sure about the default 4 for score_threshold, maybe we can force user to specify?